examples/parametric/plot_equivalent_width.py

```diff
@@ -94,5 +94,6 @@ def equivalent_width(grids, index, index_uv):
     grid_name = 'test_grid'  # Change this to the appropriate .hdf5
 
     indices = uv_indices()  # Retrieve UV indices
+    index = [i[0] for i in indices]

```

Comment:
I would much prefer not to use list comprehensions for data access on numpy arrays, and use the index notation I showed here.